### PR TITLE
[DUOS-2266][risk=no]Dataset Catalog Button Updates

### PR DESCRIPTION
**Pull request by kaitlynoflaherty**
## Addresses
https://broadworkbench.atlassian.net/jira/software/c/projects/DUOS/boards/123?modal=detail&selectedIssue=DUOS-2266

- Removes the ‘Download Dataset List’ button from Researcher views of the Dataset Catalog (i.e. only displays for DAC Chairs, Members, Signing Officials, and Admins)

- Adjusts spacing around the ‘Apply for Access’ button such that there is equal top/bottom spacing between the Catalog table and the black footer.
image

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
Changes look good 👍🏽

File: src/pages/DatasetCatalog.js [hunk 1]
```diff
@@ -513,7 +513,7 @@ export default function DatasetCatalog(props) {
 
             button({
               id: 'btn_addDataset',
-              isRendered: (currentUser.isAdmin || currentUser.isChairPerson),
+              isRendered: (currentUser.isAdmin || currentUser.isChairPerson ),
```
Comment (fboulnois) [inline src/pages/DatasetCatalog.js]: Is the space needed for formatting here?

File: src/pages/DatasetCatalog.js [hunk 2]
```diff
@@ -818,6 +818,7 @@ export default function DatasetCatalog(props) {
         div({ className: 'col-lg-5 col-md-5 col-sm-12 col-xs-12 search-wrapper no-padding' }, [
           button({
             id: 'btn_downloadSelection',
+            isRendered: (currentUser.isAdmin || currentUser.isChairPerson || currentUser.isMember || currentUser.isSigningOfficial ),
```
Comment (fboulnois) [inline src/pages/DatasetCatalog.js]: Ditto here.

**Review by fboulnois — APPROVED** [review]
lgtm, minor nits above.

**Pull request merged**